src/ec/packages/ec-component-breadcrumb-standardised/__snapshots__/breadcrumb-standardised.test.js.snap

```diff
@@ -6,12 +6,12 @@ exports[`EC - Breadcrumb Standardised Long renders correctly 1`] = `
   class="ecl-breadcrumb-standardised"
   data-ecl-auto-init="BreadcrumbStandardised"
   data-ecl-breadcrumb-standardised="true"
+  null=""
 >
   <ol
     class="ecl-breadcrumb-standardised__container"
   >
     <li
-      aria-hidden="false"
```

Comment:
it's javascript, again, handling those...

Reply:
ok but I think without any javascript ECL is setting `aria-hidden` by default to `false` in the rendered markup, no?

https://github.com/ec-europa/europa-component-library/blob/72bb2582f0a4451739a7da0a30cdd60756849c3e/src/systems/ec/implementations/react/components/breadcrumb-standardised/src/BreadcrumbStandardisedItem.jsx#L26

Reply:
but what's the value of that? i mean, we set the attribute because we want to hide or show if it's hidden, not because the attribute is needed for any other reason.